ModuleDescriptionParser/CMakeLists.txt

```diff
@@ -181,12 +180,19 @@ endif()
 
 target_link_libraries(${lib_name} ${libs})
 ## target_link_libraries(${lib_name}-static ${link_libs})
+set(USE_COMPILER_HIDDEN_VISIBILITY ON)
+if (USE_COMPILER_HIDDEN_VISIBILITY) 
```

Comment:
if `USE_COMPILER_HIDDEN_VISIBILITY` is always ON, what about removing the `if` statement ?